package.json

```diff
@@ -40,9 +40,13 @@
     "*.md"
   ],
   "devDependencies": {
-    "benchmark": "2.1.4",
-    "eslint": "8.7.0",
+    "@typescript-eslint/eslint-plugin": "^5.30.7",
+    "@typescript-eslint/parser": "^5.30.7",
+    "benchmark": "~2.1.4",
+    "eslint": "^7.16.0",
     "nyc": "^15.1.0",
-    "mocha": "5.x"
+    "mocha": "^9.2.2",
```

Comment:
Yeah let's downgrade Mocha for now to avoid having to ship a backwards breaking release. Although we should drop support for old versions of Node when we ship a new version of this package.

Reply:
what would you prefer:
- updating the commit to not update mocha
- using a separate commit to revert mocha's version?

Reply:
separate commit to revert mocha's version

Reply:
done in 14de62742b495c29fcfd1d33055dcd6863f567e4